L79-cpp-byte-buffer-slice-methods.md

```diff
@@ -0,0 +1,99 @@
+C++ API changes on ByteBuffer and Slice
+----
+* Author(s): 
+* Approver: vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2021-04-19
+* Discussion at: https://groups.google.com/g/grpc-io/c/OsAYW1mDJ9w
+
+## Abstract
+
+To better support custom serialization protocols besides protobuf, additional methods will be added to `grpc::ByteBuffer` and `grpc::Slice` those serialization protocols to access data without copying data.
```

Comment:
sounds like the proposed changes are strictly additive so the APIs change is not-breaking (and everything stays fully backward compatible). I think the proposal should explicitly say that API backwards compatibility is fully preserved (if that's the case) and that the proposed changes are non-breaking (because they are additive).

Reply:
That's a good point. I updated it.